environment.yml

```diff
@@ -2,25 +2,21 @@ name: oggm-3dviz
 channels:
   - conda-forge
 dependencies:
+  - contextily
   - ffmpeg
   - imageio=2.27.0
   - ipywidgets
   - jupyterlab
   - netcdf4
-  - pystac
-  - pystac-client
   - python>=3.10
   - pyvista
   - pyvista-xarray
   - rioxarray
   - scikit-image
-  - stackstac
   - trame
   - trame-vtk
```

Comment:
I additionally needed to add `- trame-vuetify` to get the interactive jupyter widget working